include/libtorrent/settings_pack.hpp

```diff
@@ -321,7 +321,9 @@ namespace libtorrent
 
 			// prefer seeding torrents when determining which torrents to give
 			// active slots to, the default is false which gives preference to
-			// downloading torrents
+			// downloading torrents. Be carefull if you want to prefer seeds,
```

Comment:
I think the "be careful if you want to prefer seeds, because" is redundant and can be removed.